packages/components/src/switch/src/Switch.css

```diff
@@ -1,38 +1,50 @@
 .o-ui-switch {
+    --o-ui-switch-hover-background: var(--hop-neutral-surface-hover);
+    --o-ui-switch-hover-border: var(--hop-neutral-border-hover);
+    --o-ui-switch-hover-icon: var(--hop-neutral-icon-hover);
+    --o-ui-switch-focus-background: var(--hop-neutral-surface-hover);
+    --o-ui-switch-focus-border: var(--hop-neutral-border-hover);
+    --o-ui-switch-focus-icon: var(--hop-neutral-icon-hover);
     display: inline-flex;
     align-items: center;
     cursor: pointer;
-    line-height: var(--o-ui-lh-1);
     max-width: 100%;
     width: max-content;
     height: max-content;
-    color: var(--o-ui-text-alias-primary);
+    color: var(--hop-neutral-text);
+}
+
+.o-ui-switch, .o-ui-switch::before, .o-ui-switch::after {
+    box-sizing: border-box;
 }
 
 /* SWITCH */
-.o-ui-switch .o-ui-switch-control {
+.o-ui-switch-control {
     display: inline-block;
-    background-color: var(--o-ui-bg-alias-hard-break);
-    border-radius: var(--o-ui-br-pill);
+    background-color: var(--hop-neutral-surface);
+    border: 0.06rem solid var(--hop-neutral-border);
+    border-radius: var(--hop-shape-pill);
+    box-shadow: 0 0 0 0.12rem transparent, 0 0 0 0.219rem transparent;
     position: relative;
     flex-shrink: 0;
     transition: all var(--o-ui-easing-duration-2) var(--o-ui-easing-focus);
 }
 
-.o-ui-switch .o-ui-switch-control::before {
+.o-ui-switch-control::before {
+    --o-ui-switch-control-before-transform-translate-value: var(--hop-space-inline-xs);
     content: "";
     display: inline-block;
-    background-color: var(--o-ui-bg-alias-static-white);
-    border-radius: var(--o-ui-br-circular);
+    background-color: var(--hop-neutral-icon);
```

Comment:
but it actually uses icon in the design so I tried to be consistent:
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/9180e669-16f2-4e3f-b30b-428a67eb91bd)
Probably because they are using an icon.

Reply:
I'm splitting hairs here for sure. To me it's not an icon as it's not a SVG, Design had no way to tell how we would implement this. I'm just concerned that one day we have an icon in the circle(we had it in Orbit once).

Reply:
Well considering --hop-neutral-surface is white, unless I go with the strong version, I'm going to stick with icon. It's what the figma has, it can still be considered a sort of icon. If ever we add an icon or change this somehow, it'll probably be reflected in the design anyway. We can fix it if it comes up.